app/main.py

```diff
@@ -12,3 +16,7 @@
     if animal_age < first_year + second_year:
         return 1
```

Comment:
The function `convert_to_human` should return a more meaningful value for ages less than `first_year + second_year`. Consider returning a calculated value based on the input parameters.